src/pages/patientView/genomicOverview/Tracks.tsx

```diff
@@ -112,10 +113,22 @@ export default class Tracks extends React.Component<TracksPropTypes, {}> {
 
     }
 
+    private getSvg() {
+        return document.getElementsByClassName("genomicOverviewTracksContainer")[0].firstChild as SVGElement | null;
```

Comment:
you should learn about React's ref pattern to avoid this global lookup.  See this documentation and look at "callback ref"  
https://reactjs.org/docs/refs-and-the-dom.html#callback-refs

Reply:
i'm sure you found the above code elsewhere so there may be some reason we're not doing it there.  maybe just laziness!